### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by campagnola**
WIP: filling out stream documentation, other minor docs edits

### CODE REVIEW

**Comment by samuelgarcia** [comment]
Thank you for all this.

Your english is perfect :)


tell me we can merge this.

File: pyacq/dsp/overlapfiltfilt.py [hunk 1]
```diff
@@ -390,39 +389,39 @@ def set_params(self, engine, coefficients, nb_channel, dtype, chunksize, overlap
...
     
-    The chunksize and the overlapsize are important for the accuracy of filtering.
-    You need to study them carfully, otherwise the result should be the same as a
-    real filtfilt ona long term signal. You must check the residual between real offline filtfitl
-    and this online OverlapFiltfilt.
-    Note that the chunksize have a strong effect on low frequency.
-    
-    This uses Second Order (sos) coeeficient.
-    It internally use scipy.signal.sosfilt which is available only on scipy >0.16
+    It internally uses scipy.signal.sosfilt which is available only on scipy >0.16
```
Comment (campagnola) [inline pyacq/dsp/overlapfiltfilt.py]: @samuelgarcia, check this carefully--I'm not certain I understood the original correctly.

**Comment by campagnola** [comment]
I think it's ready!

**Comment by samuelgarcia** [comment]
Now scipy 0.18 is released. So we can completely remove this scipy==0.16 warning.

**Comment by campagnola** [comment]
I totally forgot about this PR. Looks ready to merge to me..

**Comment by samuelgarcia** [comment]
I forgot too! And I have just saw that I made comments that were not submitted!! Sorry.

**Comment by campagnola** [comment]
yay!

**Pull request merged**